src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AlleleFractionModeller.java

```diff
@@ -158,10 +160,26 @@ public void fitMCMC(final int numSamples, final int numBurnIn) {
         return posteriorSummaries;
     }
 
+    /**
+     * Returns a Map of {@link PosteriorSummary} elements summarizing the global parameters.
+     * Should only be called after {@link AlleleFractionModeller#fitMCMC(int, int)} has been called.
+     * @param credibleIntervalAlpha credible-interval alpha, must be in (0, 1)
+     * @param ctx                   {@link JavaSparkContext} used for mllib kernel density estimation
+     * @return                      list of {@link PosteriorSummary} elements summarizing the global parameters
+     */
+    public Map<AlleleFractionParameter, PosteriorSummary> getGlobalParameterPosteriorSummaries(final double credibleIntervalAlpha, final JavaSparkContext ctx) {
+        final Map<AlleleFractionParameter, PosteriorSummary> posteriorSummaries = new LinkedHashMap<>();
+        posteriorSummaries.put(AlleleFractionParameter.MEAN_BIAS, PosteriorSummaryUtils.calculateHighestPosteriorDensityAndDecilesSummary(meanBiasSamples, credibleIntervalAlpha, ctx));
+        posteriorSummaries.put(AlleleFractionParameter.BIAS_VARIANCE, PosteriorSummaryUtils.calculateHighestPosteriorDensityAndDecilesSummary(biasVarianceSamples, credibleIntervalAlpha, ctx));
+        posteriorSummaries.put(AlleleFractionParameter.OUTLIER_PROBABILITY, PosteriorSummaryUtils.calculateHighestPosteriorDensityAndDecilesSummary(outlierProbabilitySamples, credibleIntervalAlpha, ctx));
+        return posteriorSummaries;
+    }
+
     //guess the width of a probability distribution given the position of its mode using a gaussian approximation
     private static double estimateWidthAtMode(final Function<Double, Double> logPDF, final double mode) {
-        final double EPSILON = Math.min(1e-6, Math.abs(mode)/2);    //adjust scale is mode is very near zero
-        final double DEFAULT = 1.0; //should never be needed; only used if mode is not a mode
+        final double absMode = Math.abs(mode);
+        final double EPSILON = Math.min(1e-6, absMode / 2);    //adjust scale is mode is very near zero
+        final double DEFAULT = absMode / 10;                   //if "mode" is not close to true mode of logPDF, approximation may not apply; just use 1/10 of absMode in this case
```

Comment:
renamed method to `estimateStepSizeFromPosteriorWidthAtMode`.  since it is a private method, i don't think it is likely to be abused in the future with this more verbose name